concepts/observability-strategy/configurable-monitoring/pocs/scrape-scalability/README.md

```diff
@@ -2,17 +2,17 @@
 
 ## Goal
 
-The agent must support scraping metrics in the prometheus pull approach. For that the [prometheusreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/prometheusreceiver) needs to be used.
+The collector must support scraping metrics in the prometheus pull approach. For that the [prometheusreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/prometheusreceiver) needs to be used.
 As the instance keeps state about the scraped data, one target cannot be scraped by interchanging instances (sticky targets we could say).
```

Comment:
```suggestion
Because the instance keeps state about the scraped data, one target cannot be scraped by interchanging instances (sticky targets, we could say).
```